src/content/learn/managing-state.md

```diff
@@ -693,15 +693,15 @@ ul, li { margin: 0; padding: 0; }
 
 <LearnMore path="/learn/extracting-state-logic-into-a-reducer">
 
-Read **[Extracting State Logic into a Reducer](/learn/extracting-state-logic-into-a-reducer)** to learn how to consolidate logic in the reducer function.
+Baca **[Mengekstrak Logika Keadaan (*State*) ke dalam Reducer](/learn/extracting-state-logic-into-a-reducer)** untuk mempelajari cara mengkonsolidasikan logika dalam fungsi reducer.
 
 </LearnMore>
 
-## Passing data deeply with context {/*passing-data-deeply-with-context*/}
+## Melewatkan data secara dalam dengan *context* {/*passing-data-deeply-with-context*/}
 
-Usually, you will pass information from a parent component to a child component via props. But passing props can become inconvenient if you need to pass some prop through many components, or if many components need the same information. Context lets the parent component make some information available to any component in the tree below it—no matter how deep it is—without passing it explicitly through props.
+Biasanya, Anda akan melewatkan informasi dari komponen induk (*parent*) ke komponen anak (*children*) melalui `props`. Namun, melewatkan `props` dapat menjadi merepotkan jika Anda perlu melewatkan beberapa `prop` melalui banyak komponen, atau jika banyak komponen membutuhkan informasi yang sama. *Context* memungkinkan komponen induk membuat beberapa informasi tersedia untuk setiap komponen di bawahnya—tidak peduli seberapa dalam itu—tanpa melewatkan secara eksplisit melalui `props`.
```

Comment:
```suggestion
Biasanya, Anda akan melewatkan informasi dari komponen induk ke komponen anak (*children*) melalui `props`. Namun, melewatkan `props` dapat menjadi merepotkan jika Anda perlu melewatkan beberapa `prop` melalui banyak komponen, atau jika banyak komponen membutuhkan informasi yang sama. *Context* memungkinkan komponen induk membuat beberapa informasi tersedia untuk setiap komponen di bawahnya—tidak peduli seberapa dalam itu—tanpa melewatkan secara eksplisit melalui `props`.
```

According to [this writing guide](https://github.com/reactjs/id.react.dev/wiki/Panduan-Penulisan-Universal#2-memperkenalkan-kata-baru-dalam-bahasa-indonesia-sambil-mengutip-kata-aslinya), once you introduced "induk (*parent*)" translation for the first time above (at line `234`), you don't need to repeat it for the subsequent occurrences. So you may leave the "(*parent*)" part out.

![image](https://user-images.githubusercontent.com/6315466/236003385-6fe53473-bd91-427f-ab51-11acbef8de1a.png)